OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -35,6 +35,7 @@
     <Reference Include="System" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="OpenRA.Game\IntersectTests.cs" />
```

Comment:
This is failing AppVeyor. Either remove this line or add the `IntersectTests.cs` file.